### PR TITLE
Midnight happenings

### PR DESCRIPTION
**Pull request by EnnuiL**
midnight happenings

### CODE REVIEW

File: mappings/net/minecraft/client/particle/CherryLeafParticle.mapping [hunk 1]
```diff
@@ -1 +1,8 @@
 CLASS net/minecraft/unmapped/C_ewetkmcp net/minecraft/client/particle/CherryLeafParticle
+	FIELD f_tmcykyqu rotationSpeed F
+	METHOD <init> (Lnet/minecraft/unmapped/C_ghdnlrrw;DDDLnet/minecraft/unmapped/C_audtngae;)V
+		ARG 1 world
+		ARG 2 x
+		ARG 4 y
+		ARG 6 z
+		ARG 8 SpriteProvider
```
Comment (IotaBread) [inline mappings/net/minecraft/client/particle/CherryLeafParticle.mapping]:
```suggestion
		ARG 8 spriteProvider
```

**Review by IotaBread — APPROVED** [review]

File: mappings/net/minecraft/client/particle/CherryLeafParticle.mapping (hunk 1 above)
Comment (EnnuiL) [reply mappings/net/minecraft/client/particle/CherryLeafParticle.mapping]: yes am already on the way

**Review by supersaiyansubtlety — APPROVED** [review]

**Review by eristhea — CHANGES_REQUESTED** [review]
I'm not a big fan of the word order on `velocityXYZ` parameters, I'd prefer we went with the natural `xyzVelocity`

**Comment by EnnuiL** [comment]
> I'm not a big fan of the word order on `velocityXYZ` parameters, I'd prefer we went with the natural `xyzVelocity`

The problem is that it goes against [the convention](https://github.com/QuiltMC/quilt-mappings/blob/1.20.1/CONVENTIONS.md#coordinates) about that

**Comment by eristhea** [comment]
I'd argue that `xyzVelocity` fits the convention better -- you wouldn't say "this updates the velocity X of the entity to 5 blocks per second", the more natural sentence is "updates the x velocity"

**Comment by EnnuiL** [comment]
> I'd argue that `xyzVelocity` fits the convention better -- you wouldn't say "this updates the velocity X of the entity to 5 blocks per second", the more natural sentence is "updates the x velocity"

I'd argue that if you think of it as "velocity's X", things end making more sense; But oh well, I feel like a convention update/mass refactor would be a separate PR

**Comment by eristhea** [comment]
ok, I'll put it on my todo list :p

**Review by eristhea — APPROVED** [review]

**Pull request merged**